BranchMetrics/android-branch-deep-linking-attribution
# Sdk 953 branch json

## Reference
SDK-953 -- add branch.json configuration to Android SDK.

## Description
A new way to set various configurations via a json file stored in assets.

## Testing Instructions
Add branch.json file to assets in testbed app as per format seen in the comments in `BranchJsonConfig` and observe your configuration for, say, `enableFacebookAppLinkCheck` take place without client having to call 
`Branch.getInstance().enableFacebookAppLinkCheck();`

## Risk Assessment [`HIGH` || `MEDIUM` || `LOW`]
MEDIUM


## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style
